web_client/stylesheets/panels/annotationSelector.styl

```diff
@@ -9,7 +9,7 @@
     background none
 
   input
-    width 210px
+    width 200px
```

Comment:
In the current version of Chrome I'm using, 200px still wraps (but 199px doesn't).

Reply:
I changed it to 195.  I'm sure there is a better way to do this in css...